client/src/components/ProtectedRoutes/ProtectedRoutes.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Redirect, Route, RouteProps } from 'react-router-dom';
+import { useAuth } from '../../context/useAuthContext';
+
+const ProtectedRoutes: React.FC<RouteProps> = ({ ...rest }) => {
+  const auth = useAuth();
+  if (auth.loggedInUser === null || auth.loggedInUser === undefined) return <Redirect to="/login" />;
```

Comment:
This test can be written like this: `if (!auth.loggedInUser)` since both `null` and `undefined` are Falsy.